lib/kadai-core/src/main/java/io/kadai/spi/routing/internal/TaskRoutingManager.java

```diff
@@ -90,6 +91,42 @@ public String determineWorkbasketId(Task task) {
     return workbasketId;
   }
 
+  /**
+   * Determines a {@linkplain RoutingTarget} for a given {@linkplain Task}.
+   * Algorithm: see {@linkplain TaskRoutingManager#determineWorkbasketId(Task)}.
+   *
+   * @param task the {@linkplain Task} for which a {@linkplain RoutingTarget} is to be determined
+   * @return the {@linkplain RoutingTarget} for the creation of the {@linkplain Task}
+   */
+  public RoutingTarget determineRoutingTarget(Task task) {
```

Comment:
I'm curious: what happens if we have an old RoutingProvider which implements the old interface with only one operation? Do we need versioning?

Reply:
This method only gets called if kadai.routing.includeOwner=true is set. In this case, only the method determineRoutingTarget of the RoutingProvider is relevant. If the RoutingProvider doesn't implement such method, then the default method implementation is called. The default method implementation returns an empty RoutingTarget. Then, a suitable error is thrown. So I don't see a problem here. Would you agree?